Assets/HealthUI.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthUI : MonoBehaviour
+{
+    [SerializeField] private GameObject[] dragonHealthUI;
+    [SerializeField] private GameObject[] knightHealthUI;
+    [SerializeField] private int dragonHealth;
+    [SerializeField] private int knightHealth;
+    // Start is called before the first frame update
+    void Start()
+    {
+        foreach (GameObject life in dragonHealthUI)
+        {
+            life.SetActive(true);
+        }
+        foreach (GameObject life in knightHealthUI)
+        {
+            life.SetActive(true);
+        }
```

Comment:
You could enable the hearts in the scene editor, which would remove the need to set them to active on game start here.